user-guide/Cloud_Platform/AboutCloudPlatform/Custom_cloud_endpoint_configuration.md

```diff
@@ -0,0 +1,32 @@
+---
+uid: Custom_cloud_endpoint_configuration
+---
+
+# Customizing the cloud endpoint configuration
+
+From version 2.10.0 of the [DataMiner CloudGateway](xref:DataMinerExtensionModules#cloudgateway) DataMiner Extension Module (DxM) onwards, this DxM hosts an HTTP(S) endpoint that allows other [DxMs](xref:DataMinerExtensionModules) to communicate with the cloud via DataMiner CloudGateway in an efficient and secure way without the need for direct internet access.
+
+This endpoint is required for some features, such as [Remote Log Collection](xref:RemoteLogCollection). It only allows traffic to *\*.dataminer.services*, which you can see in the default *CloudEndpointOptions* configuration under *FirewallRules*.
```

Comment:
", which you can see in the default *CloudEndpointOptions* configuration under *FirewallRules*." => This is a bit vague. Where can the user find "FirewallRules"? And even so, is it necessary to add this? It might be better to keep things simple and just state "It only allows traffic to *\*.dataminer.services*.".

Reply:
It's about the appsettings. We could leave this away but adjust other places to be clear it is for 'communication with the DataMiner Cloud Platform' instead of just saying 'communicate with the cloud' ?

Reply:
If you agree i can change that :)